### PR TITLE
LPS-198626 Format properly the DATE_TIME field, when DBType is ORACLE

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-198626

### CODE REVIEW

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/filter/expression/PredicateExpressionVisitorImpl.java [hunk 1]
```diff
@@ -630,11 +630,18 @@ private Object _getValue(
...
 
 		if (entityType.equals(EntityField.Type.DATE_TIME) &&
-			(DBManagerUtil.getDBType() == DBType.HYPERSONIC)) {
+			(Objects.equals(DBManagerUtil.getDBType(), DBType.HYPERSONIC) ||
+			 Objects.equals(DBManagerUtil.getDBType(), DBType.ORACLE))) {
+
+			String pattern = "dd-MMM-yyyy HH:mm:ss.SSS";
+
+			if (Objects.equals(DBManagerUtil.getDBType(), DBType.ORACLE)) {
+				pattern = StringUtil.replace(pattern, "HH", "hh") + " a";
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/filter/expression/PredicateExpressionVisitorImpl.java:639]: @p-albuquerque I would change to use the string instead, like: `pattern = "dd-MMM-yyyy hh:mm:ss.SSS a";`

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**